README.md

```diff
@@ -9,10 +9,45 @@
 
 Icepeak is a fast JSON document store with push notification support.
 
-## Disclaimer
+Icepeak has an HTTP API that can be used to both read and write data.
+It also has a websocket interface that clients can use to get push updates about
+data that changed.
 
-This is alpha-quality software developed under Hackathon conditions.
-Do not use it.
+Clients can subscribe to specific paths in the JSON document like e.g. `/users/123/status`
+via the websocket-based API and they will get a push update whenever any of the
+JSON data below this path changes.
+
+If a client subscribes to the document root `/` they will receive *all* updates.
+
+Icepeak supports JWT-based authorization and authentication. Authentication can be used
```

Comment:
We're only doing authorization in Icepeak, right? Authentication is done by whoever is handing out the tokens.

Reply:
Good point, I'll update this.